### DISCUSSION TITLE
SES Email Verified but no magic link emails being sent

### DISCUSSION THREAD

**Discussion by github-user-77** [mozilla/hubs#3274] [Hubs Cloud Questions] [answered]
Hi There,
I had no problem getting email to my administrator account, and getting magic link for it. Then had no problem getting other email addresses verified by SES, but those addresses are not receiving magic links. The hubs page does say "email sent to" with the correct address, but it never arrives. Also, the sent email count is zero in SES, even though obviously it has sent the verification emails (or do they not count in the sent total?).

I believe I am looking in the correct region, i.e., the region that I deployed HC in, but I checked in N.Virginia (just in case!) but same thing. Not sure what to do from here. Any help most appreciated. I'd prefer to keep sign-in ability restricted to registered email addresses, rather than open it up by requesting rate increase from AWS.

**Comment by elmau** [comment]
Maybe you're limited by AWS, use a different email settings in the admin area. That worked for me. There's another solution on the documentation also, but I haven't tried it.

**Reply by github-user-77** [reply]
Thanks @elmau, seems unlikely to be a limit problem, as it's only been 3 emails. I was wanting to avoid using other email settings because I only want verified email addresses to be able to sign-in and/or create rooms. 

Perhaps I am misunderstanding this part of the functionality in Hubs Cloud, but it seems that if I use #2 "Send request to AWS for limit increase", or "#3 Setup SMTP via SendGrid", then any address can sign-in and create rooms, so I think I want to stick with "#1 Verify individual email addresses for administrators", in order that only a certain, approved group of people can create rooms.

**Reply by elmau** [reply]
You can disable room creation for users, even if they authenticate.

**Reply by github-user-77** [reply]
Thanks for replying @elmau, is that for all users though? I'm trying to figure out how to have a list of email addresses that can create rooms, where everyone else cannot.

**Reply by github-user-77** [reply]
Ok, I got a SendGrid API Key and used that in the email settings as per #3, but still no magic link emails being sent. I wonder what I am doing wrong. Next, I'll try requesting limit increase from AWS I guess.

**Reply by github-user-77** [reply]
So now I've received limit increase from AWS, sending quota now 50,000 emails per day,  max send rate 14 emails per second, and my account has been moved out of the SES sandbox. I've deleted all the SMTP info in Admin Panel –  Server Settings – Email (all settings are now blank).

Magic link emails are still NOT being sent. 

I don't know what to do now. Tearing my hair out. It seems so simple, but why is it not working??

**Reply by elmau** [reply]
No idea how that works. I used a different smtp settings from another domain that I have, and the mails are going out without problems.

**Reply by github-user-77** [reply]
@elmau, so you just used a regular smtp server on a different domain, and you put those details in the Admin panel in Server Settings - Email?

**Reply by github-user-77** [reply]
Update: my Hubs Cloud instance will send a magic link email to my administrator address (i.e., my hub stack's admin email address), but will not send a magic link email to any other email address. (This is with using the AWS email, rate limit increased to 50K and out of sandbox.) This is driving me crazy, as none of my users can log in, and I just can't figure out what I'm doing wrong.

**Comment by github-user-77** [comment, marked as answer]
SOLVED: my Hubs Cloud instance is now sending magic links correctly. I am not sure why, but I assume it took some time for the SMTP settings to take effect. Thanks for the help everybody, especially @elmau, and apologies for flooding the channel with my silly problems!

**Reply by MegaMotion** [reply]
Hi, I seem to be having the same problem you had, I was just wondering how long it took for your issue to go away? I was released from the sandbox a day or two ago, and just tonight I realized I had not given it my domain name as a verified sender domain, so I did that, and it says "Verified". However, I still do not see any emails, after a couple of hours.
